uber/custom_tags.py

```diff
@@ -17,23 +17,31 @@ def safe_string(text):
 
 @JinjaEnv.jinja_filter(name='datetime')
 def datetime_filter(dt, fmt='%-I:%M%p %Z on %A, %b %e'):
-    return ' '.join(dt.strftime(fmt).split()).replace('AM', 'am').replace('PM', 'pm') if dt else ''
+    if not dt:
+        return ''
```

Comment:
`datetime_filter` was actually already checking for an empty argument using the ternary `if` `else` conditional. However, I find the explicit check at the top of the function to be more obvious/readable.

In fact, the root cause of the issue was that I didn't notice the ternary conditional when I was writing the similar `datetime_local_filter` function below.